README.rst

```diff
@@ -26,39 +26,56 @@
  plotextractor
 ===============
 
-Small library for converting and mapping plots in TeX source used in scholarly communication.
+.. image:: https://travis-ci.org/inspirehep/plotextractor.svg?branch=master
+    :target: https://travis-ci.org/inspirehep/plotextractor
 
-* Free software: GPLv2
-* Documentation: http://pythonhosted.org/plotextractor/
```

Comment:
This link to the docs has disappeared, is that on purpose?

Reply:
PythonHosted is dead (https://github.com/pypa/pypi-legacy/issues/700#issuecomment-378470711), that documentation will be deleted as well as the link.